addons/addon-base-raas/packages/base-raas-services/lib/study/__tests__/study-service.test.js

```diff
@@ -251,6 +251,50 @@ describe('studyService', () => {
       );
     });
 
+    it('should fail to update resource list of non-Open Data study', async () => {
+      // BUILD
+      const dataIpt = {
+        id: 'newOpenStudy',
+        category: 'Organization',
+        resources: [{ arn: 'arn:aws:s3:::someRandomStudyArn' }],
+      };
+      service.audit = jest.fn();
+
+      // OPERATE
+      try {
+        await service.update(
+          { principal: { userRole: 'researcher' }, principalIdentifier: { uid: 'someRandomUserUid' } },
+          dataIpt,
+        );
+        expect.hasAssertions();
+      } catch (err) {
+        // CHECK
+        expect(err.message).toEqual('Resources can only be updated for Open Data study category');
```

Comment:
For future tests, you can use expect(<>).rejects.toThrow(..)

Reply:
Will do. Thanks